Multisig/UI/App/UIKit/Settings/SafeSettingsViewController/SafeSettingsViewController.swift

```diff
@@ -256,8 +256,8 @@ class SafeSettingsViewController: LoadableViewController, UITableViewDelegate, U
         let item = sections[indexPath.section].items[indexPath.row]
         switch item {
         case Section.Name.name(_):
-            let vc = ViewControllerFactory.editSafeNameController(address: safe.address, name: safe.name, presenter: self)
-            present(vc, animated: true, completion: nil)
+            let vc = ViewControllerFactory.editSafeNameController(address: safe.addressValue, name: safe.name, presenter: self)
+            show(vc, sender: self)
```

Comment:
why don't we just create the edit safe name view controller directly here? The previous "viewcontrollerfactory" was there as a way to put all swiftui-related controller creation in one place

Reply:
I wanted to keep the consistency of our code base, but I will change it